examples/tuned_gbdt.py

```diff
@@ -88,6 +89,11 @@
     gbdt.tune(tf_train=train_dataset.tensor_frame,
               tf_val=val_dataset.tensor_frame, num_trials=20)
     gbdt.save(args.saved_model_path)
+    scores = pd.DataFrame({
+        'feature': dataset.feat_cols,
+        'importance': gbdt.feature_importance()
+    }).sort_values(by='importance', ascending=False)
+    print(scores)
```

Comment:
Add some more text around the `scores`

Reply:
Maybe we can add an parser argument to enable user specify whether they want to have feature importance.